test/templateCount.test.ts

```diff
@@ -45,9 +45,13 @@ function addSuite(source: TemplateSource | undefined): void {
             { language: ProjectLanguage.PowerShell, version: FuncVersion.v2, expectedCount: 14 },
             { language: ProjectLanguage.PowerShell, version: FuncVersion.v3, expectedCount: 14 },
             { language: ProjectLanguage.PowerShell, version: FuncVersion.v4, expectedCount: 14 },
-            { language: ProjectLanguage.Java, version: FuncVersion.v2, expectedCount: 4 }
+            { language: ProjectLanguage.Java, version: FuncVersion.v2, expectedCount: 4 },
             // https://github.com/microsoft/vscode-azurefunctions/issues/1605
             // { language: ProjectLanguage.Java, version: FuncVersion.v3, expectedCount: 4}]
+            { language: ProjectLanguage.Ballerina, version: FuncVersion.v1, expectedCount: 5 },
```

Comment:
Are you actually creating templates for all runtimes of functions? I believe v2 and v3 are deprecated or EOL, so we probably don't need to add new tests for that.